.envrc

```diff
@@ -0,0 +1,2 @@
+[[ -f .envrc.local ]] && source_env .envrc.local
+use flake
```

Comment:
What does this do?

Reply:
`direnv` allows to load the flake when `cd`ing into the directory. It's very convenient, but of course I shouldn't have smuggled it inside this PR :D Happy to remove it and put it into a follow up PR on the flakes PR (which I'll look at soon).